chainer_chemistry/functions/activation/shifted_softplus.py

```diff
@@ -0,0 +1,15 @@
+import chainer
+from chainer import functions
+
+
+def shifted_softplus(x, beta=0.5):
```

Comment:
Can you use `beta` as the argument of `softplus` function and please introduce new value `shift` for shift purpose.
Please refer https://docs.chainer.org/en/stable/reference/generated/chainer.functions.softplus.html

Reply:
https://github.com/dmlc/dgl/blob/master/python/dgl/model_zoo/chem/layers.py#L128-L147